keyboards/sneakbox/disarray/staggered/disarray.c

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 Bryan Ong
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "disarray.h"
+
+void keyboard_pre_init_kb() {
+    setPinOutput(B1);
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    writePin(B1, 1);
+    return res;
+}
```

Comment:
Same as above.

Reply:
Thanks. I made the change to migrate to #define in config.h